<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Produit</class>
 <widget class="QMainWindow" name="Produit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fewnu</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/img/logo.png</normaloff>:/img/img/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#frame{
 border-image:url(:/img/img/profile.jpg);
position:fixed;
border-radius: 40px;
}



#gridLayout_2{

   background-color: #ffffff;
}



#frame_2{
 border-image:url(:/img/img/logo.png);
position:fixed;
}

#groupBox_2,#groupBox_3,#line{

  background-color: #008ea1;
 border: 0px;
}

#groupBox,#horizontalSpacer,#horizontalSpacer_3,#horizontalSpacer_4,#horizontalSpacer_2{

    background-color: #008ea1;

}
#groupBox_4,#groupBox_5,#groupBox_6,#groupBox_7,#groupBox_8,#groupBox_9,#groupBox_10{
   
 background-color: #ffffff;



}



#label,#label_2 ,#label_3,#label_4,#label_6,#label_7,#label_8,#label_9,#label_10,#label_11{
       
 color: #ffffff; 
 margin-top: 0;
 margin-bottom: 1rem

}




#pushButton_profile,#pushButton_deconnrctet,#pushButton_achats,#pushButton_pro,#Retour,#calculator,#power
{
    border: 2px solid #008ea1;
    padding: 5px;
   border-radius: 4px;
 
    border: 2px solid #ffffff;
     background-color: #008ea1;
 
}

#pushButton_profile:hover { background-color: #4c8dff; }
#pushButton_deconnrctet:hover { background-color: #4c8dff; }

#groupBox_5{

  border: 2px solid #008ea1;
    padding: 5px;
   border-radius: 20px;
 
    border: 2px solid #ffffff;
     background-color: #008ea1;
}



#pushButton_Ajoute
{
    border: 2px solid #008ea1;
    padding: 5px;
   border-radius: 20px;
 
    border: 0px solid #ffffff;
     background-color: #ffc107;
 
}

#Input_DES,#Input_Q,#Input_P{
       border: 1px solid #f4f5f8;
}

#pushButton_Valider{
       border: 2px solid #008ea1;
    padding: 5px;
   border-radius: 4px;
  color: #ffffff; 
    border: 2px solid #ffffff;
     background-color: #008ea1;

}
#pushButton_Valider:hover { background-color: #4c8dff; }

#groupBox_11{

  background-color: #ffffff;

}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>1051</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1111</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Retour">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/img/img/return-up-back-sharp.svg</normaloff>:/img/img/return-up-back-sharp.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>  Ajout Produit</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>1051</width>
      <height>691</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Produit#</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Input_DES">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>971</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string> Designation</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Input_Q">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>971</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string> Qantit√©</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Input_P">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>971</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string> Prix Unitaire</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Valider">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>370</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>VALIDER</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>321</width>
      <height>481</height>
     </rect>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="autoScrollMargin">
     <number>16</number>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerItem</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerItem</enum>
    </property>
    <property name="movement">
     <enum>QListView::Static</enum>
    </property>
    <property name="flow">
     <enum>QListView::TopToBottom</enum>
    </property>
    <property name="resizeMode">
     <enum>QListView::Fixed</enum>
    </property>
    <property name="layoutMode">
     <enum>QListView::SinglePass</enum>
    </property>
    <property name="spacing">
     <number>4</number>
    </property>
    <item>
     <property name="text">
      <string>ACCUEIL</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/img/home-outline.svg</normaloff>:/img/img/home-outline.svg</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Menu Principal</string>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="background">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>142</green>
        <blue>161</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MES BOUTIQUES </string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/img/newspaper-outline.svg</normaloff>:/img/img/newspaper-outline.svg</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FOURNISSEURS</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/img/cube.svg</normaloff>:/img/img/cube.svg</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VENTES</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/img/cart-outline.svg</normaloff>:/img/img/cart-outline.svg</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>STOCK</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/img/reader-outline.svg</normaloff>:/img/img/reader-outline.svg</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DEPENSES</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/img/logo-closed-captioning.svg</normaloff>:/img/img/logo-closed-captioning.svg</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PRETS</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/img/logo-usd.svg</normaloff>:/img/img/logo-usd.svg</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CLIENTS</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/img/people-outline.svg</normaloff>:/img/img/people-outline.svg</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BILANS</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/img/today-outline.svg</normaloff>:/img/img/today-outline.svg</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BENEFICE</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/img/logo-closed-captioning.svg</normaloff>:/img/img/logo-closed-captioning.svg</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SAUVEGARDER VOS DONNEES</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/img/download-outline.svg</normaloff>:/img/img/download-outline.svg</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A PROPOS</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="HorPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/img/information-circle-outline.svg</normaloff>:/img/img/information-circle-outline.svg</iconset>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBox_11">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>720</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_profile">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/img/person-outline.svg</normaloff>:/img/img/person-outline.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_deconnrctet">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>130</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/img/enter-outline.svg</normaloff>:/img/img/enter-outline.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>          name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Voir mon profil</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Se d√©connecter</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
